be/src/common/config.h

```diff
@@ -1040,8 +1040,8 @@ CONF_mInt64(max_allow_pindex_l2_num, "5");
 CONF_mInt64(pindex_major_compaction_num_threads, "0");
 // control the persistent index schedule compaction interval
 CONF_mInt64(pindex_major_compaction_schedule_interval_seconds, "15");
-// control the local persistent index in shared_data gc interval
-CONF_mInt64(pindex_shard_data_gc_interval_seconds, "18000"); // 5 hour
+// control the local persistent index in shared_data gc/evict interval
+CONF_mInt64(pindex_shared_data_gc_evict_interval_seconds, "18000"); // 5 hour
```

Comment:
have a check if we have document exposed with this typo.

Reply:
https://github.com/search?q=repo%3AStarRocks%2Fstarrocks+shard_data&type=code